### PR TITLE
OBS

### PR DESCRIPTION
**Pull request by navdeepsinghkhalsa**

### CODE REVIEW

**Comment by bogas04** [comment]
What is this feature about?

File: package.json [hunk 1]
```diff
@@ -68,7 +74,9 @@
     "request": "^2.81.0",
     "request-progress": "^3.0.0",
     "scroll": "^2.0.1",
-    "sqlite3": "^3.1.13",
+    "socket.io": "^2.1.1",
+    "sqlite3": "^3.1.8",
+    "tippy.js": "^2.5.2",
```
Comment (tsingh777) [inline package.json]: where are we using this?

File: www/js/overlay.js [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    bottomLayoutApply();
+    break;
+  case 'split':
+    splitLayoutApply();
+    break;
+  default :
+    break;
+}
+updateContentBarColor(overlayVars.bgColor, 'backgroundColor');
+updateContentBarColor(overlayVars.textColor, 'color');
```
Comment (tsingh777) [inline www/js/overlay.js]: Not related to this line but remove the `overlay.json` file?

File: www/obs/socket.io.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+/*!
+ * Socket.IO v2.1.1
```
Comment (tsingh777) [inline www/obs/socket.io.js]: We can probably use this for sync? Nothing to change right now but clean up in the future. cc @bogas04

Comment (bogas04) [reply www/obs/socket.io.js]: Yeah we can use it, I dynamically load socket.io lib from our server as that takes care of connection and everything. We can definitely use the library and connect manually

**Pull request merged**